resources/META-INF/plugin.xml

```diff
@@ -21,6 +21,17 @@
 
   <change-notes>
     <![CDATA[
+
+0.12.0:
```

Comment:
Yeah, +1. @mit-mit had some ideas around the versioning, but without other guidance let's go with `12.0`.